### PR TITLE
travis: add 9, remove 7

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/Level/level-rocksdb/issues/24 and #25

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,9 @@ addons:
       - g++-4.8
 
 before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then export CXX=g++-4.8; fi
   - export JOBS=max
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then export JOBS=4; fi
```
[the remark this answers is not in the document]
Comment (vweevers) [reply .travis.yml]: Not sure I follow. Is there an issue that provides context?

[the remark this answers is not in the document]
Comment (ralphtheninja) [reply .travis.yml]: https://github.com/Level/rocksdb/pull/3

[the remark this answers is not in the document]
Comment (ralphtheninja) [reply .travis.yml]: Will solve https://github.com/Level/level-rocksdb/issues/25

[the remark this answers is not in the document]
Comment (vweevers) [reply .travis.yml]: Thanks!

**Pull request merged**